pkgs/by-name/ir/ironbar/package.nix

```diff
@@ -43,8 +49,10 @@ rustPlatform.buildRustPackage rec {
     hicolor-icon-theme
     gsettings-desktop-schemas
     libxkbcommon
-    openssl
-  ];
+    libpulseaudio
```

Comment:
`libpulseaudio` is here twice - only need the `hasFeature` entry below.

Reply:
Idk how I didn't spot this...

Reply:
Thanks for the spot, fixed in https://github.com/NixOS/nixpkgs/pull/307726/commits/8d31eb6ea2828b5ca755869f995556c069af6ec4